.github/workflows/e2e.yaml

```diff
@@ -34,6 +34,9 @@ jobs:
         source ../venv
         drenv delete --name-prefix ${{ env.NAME_PREFIX }} envs/regional-dr.yaml
 
+    - name: Setup libvirt
+      run: test/scripts/setup-libvirt
+
```

Comment:
The github runner user cannot run this script. It doesn't have the root permissions.

I will setup the github runner to periodically setup the libvirt network.

I think we should remove this commit.

Reply:
This does not need root - the user running minikube must be in the libvirt group (per our docs), so it can run this command. This cannot be fixed by running the script on the host periodically since the network may be deleted when deleting leftover clusters.